sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-batch-scoring/scripts/batch_scoring.py

```diff
@@ -40,11 +24,9 @@ def init():
     # Set up logging
     _set_logging_parameters(TASK_TYPE, {})
 
-    parser = argparse.ArgumentParser(description="Retrieve batch_size from arguments.")
-    parser.add_argument("--batch_size", dest="batch_size", type=int, required=False)
-    args, _ = parser.parse_known_args()
-
-    batch_size = args.batch_size
+    batch_size = os.getenv('batch_size', None)
+    batch_size = int(batch_size) if batch_size is not None else batch_size
```

Comment:
just for my information, does inference use default batch size if batch_size is None?

Reply:
Yes Gaurav, that's correct